torchtune/models/qwen2/_tokenizer.py

```diff
@@ -363,7 +366,7 @@ def tokenize_messages(
             if messages:
                 last_message_masked = messages[-1].masked
             mask = mask + [last_message_masked]
-        if max_seq_len:
-            tokens = truncate(tokens, max_seq_len, self.eos_id)
-            mask = truncate(mask, max_seq_len, True)
+        if self.max_seq_len:
+            tokens = truncate(tokens, self.max_seq_len, self.eos_id)
+            mask = truncate(mask, self.max_seq_len, True)
```

Comment:
This one doesn't need to add `__call__`/ `Transform`?

Reply:
oh good call out